### DISCUSSION TITLE
How can I hide the TST header?

### DISCUSSION THREAD

**Discussion by mfn** [piroor/treestyletab#2982] [Q&A] [answered]
Apologies if this has been asked, couldn't find it in issues/discussion nor something in the prefs; I would like to remove this part outlined in red:
![image](https://user-images.githubusercontent.com/87493/129320051-1814e322-105d-4e77-9eb6-3269e8a4e5db.png)

In fact, I have not see this in a long time but when I restarted FF into 91.0 yesterday, it appeared. The `X` closes the whole TST, not just the header 😅

Thank you!

**Comment by irvinm** [comment, marked as answer]
The sidebar header can be modified via userChrome.css.  What code were you using previously to hide it?  Are your other userChrome.css changes still working?

**Reply by mfn** [reply]
> The sidebar header can be modified via userChrome.css

Hmm, I don't have any and I don't recall it having TST modified through this.

I also don't have other add-ons modifying user styles.

All I know is that from one restart to next, this big header suddently appeared where none was before (for … months? … at least a long time)

**Reply by irvinm** [reply]
Well, that is interesting.  Regardless, I think userChrome.css you are looking for is here: https://github.com/piroor/treestyletab/wiki/Code-snippets-for-custom-style-rules#hide-the-tree-style-tab-header-at-the-top-of-the-sidebar

**Reply by mfn** [reply]
Thank you, this worked like a charm!
